gatsby-config.js

```diff
@@ -125,7 +125,7 @@ module.exports = {
           },
           {
             title: 'Cloud Manager Events',
-            path: '/experience-cloud/cloud-manager/guides/getting-started/create-event-integration/'
+            path: 'https://www.adobe.io/apis/experiencecloud/cloud-manager/docs.html#!AdobeDocs/cloudmanager-api-docs/master/create-event-integration.md'
```

Comment:
Why was this path changed to the old URL which will be decommissioned soon ?

Reply:
@icaraps Ok but this relative path does not exist in this repo, so how will it get resolved?

Reply:
It's resolved once it's on stage or prod

Reply:
Ok. I have to push another documentation related PR next week so i'll take care of this.